WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/DashboardCardModel.swift

```diff
@@ -20,19 +21,22 @@ struct DashboardCardModel: Hashable {
                 view. The `hashableDictionary` and the `id` is used to differentiate one
                 card from the other.
     */
-    init(id: DashboardCard, hashableDictionary: NSDictionary? = nil, entity: BlogDashboardRemoteEntity? = nil) {
+    init(id: DashboardCard, wpComID: Int, hashableDictionary: NSDictionary? = nil, entity: BlogDashboardRemoteEntity? = nil) {
```

Comment:
It would be good to add the new property to the comment/doc above.

Reply:
Fixed! 7c21448